source/github-helpers/index.ts

```diff
@@ -68,6 +68,11 @@ export const getRepositoryInfo = (repoUrl: string = location.pathname.slice(1)):
 	return {owner, name};
 };
 
+export const getPRRepositoryInfo = (): RepositoryInfo => {
+	const [, owner, name] = select<HTMLAnchorElement>('.commit-ref.head-ref a')!.pathname.split('/', 3);
+	return {owner, name};
```

Comment:
`getRepositoryInfo` is flexible, let's reuse that:

```suggestion
	const {pathname} = select<HTMLAnchorElement>('.commit-ref.head-ref a')!;
	return getRepositoryInfo(pathname.slice(1));
```

This also enables something you can implement later:

# `lintideas`

1. Have `getRepositoryInfo` also return `url` as:
	```js
	return {owner, name, url: owner + '/' + name}
	```
    `url` is actually how this information is used most of the time anyway.
2. Replace `getRepoURL()` with `getRepositoryInfo().url`